.sync.yml

```diff
@@ -21,9 +21,6 @@
   default_configs:
     inherit_from: .rubocop_todo.yml
 
-.travis.yml:
-  unmanaged: true
```

Comment:
we have to set this also to `delete: true`, otherwise it will be recreated on the next `pdk update` run